README.md

```diff
@@ -2,9 +2,13 @@
 ## A telescope observation simulator for Python
 
 [![Build Status](https://github.com/AstarVienna/ScopeSim/actions/workflows/tests.yml/badge.svg)](https://github.com/AstarVienna/ScopeSim/actions/workflows/tests.yml/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/scopesim/badge/?version=latest)](https://scopesim.readthedocs.io/en/latest/?badge=latest)
+[![Build Status](https://github.com/AstarVienna/ScopeSim/actions/workflows/minimumdependencies.yml/badge.svg)](https://github.com/AstarVienna/ScopeSim/actions/workflows/minimumdependencies.yml/badge.svg)
+[![Build Status](https://github.com/AstarVienna/ScopeSim/actions/workflows/notebooks_with_irdb_download.yml/badge.svg)](https://github.com/AstarVienna/ScopeSim/actions/workflows/notebooks_with_irdb_download.yml/badge.svg)
 
-[![Build Status](http://github-actions.40ants.com/AstarVienna/ScopeSim/matrix.svg)](https://github.com/AstarVienna/ScopeSim)
+
+[![Documentation Status](https://readthedocs.org/projects/scopesim/badge/?version=latest)](https://scopesim.readthedocs.io/en/latest)
+[![codecov](https://codecov.io/gh/AstarVienna/ScopeSim/graph/badge.svg?token=CHU4U1RYMD)](https://codecov.io/gh/AstarVienna/ScopeSim)
```

Comment:
Is the token necessary? Seems to work without the token as well

Reply:
Copied the markdown snippet from: https://app.codecov.io/gh/AstarVienna/ScopeSim_Templates/settings/badge (login required)

Not sure why it's there though. As you pointed out, seems to work fine without it, so might as well remove it.